src/csharp/Grpc.Core/Channel.cs

```diff
@@ -221,7 +221,7 @@ public async Task ConnectAsync(DateTime? deadline = null)
         /// before shutting down the channel to ensure channel shutdown won't impact
         /// the outcome of those remote calls.
         /// </remarks>
-        public async Task ShutdownAsync()
+        public new async Task ShutdownAsync()
```

Comment:
I believe we should be able to remove this and move the logic to ShutdownAsyncCore. Any current calls to Channel.ShutdownAsync should go to ChannelBase.ShutdownAsync.

Reply:
https://docs.microsoft.com/en-us/dotnet/core/compatibility/

> ✔️ Moving a member into a class higher in the hierarchy than the type from which it was removed

Reply:
Good idea. Done.